doc/using_mxnet.rst

```diff
@@ -1,822 +1,784 @@
+
+Suppose that you already have an MXNet training script called
+``mxnet-train.py``. You can run this script in SageMaker as follows:
+
+.. code:: python
+
+    from sagemaker.mxnet import MXNet
+    mxnet_estimator = MXNet('mxnet-train.py',
+                            role='SageMakerRole',
+                            train_instance_type='ml.p3.2xlarge',
+                            train_instance_count=1,
+                            framework_version='1.2.1')
+    mxnet_estimator.fit('s3://bucket/path/to/training/data')
+
+Where the S3 url is a path to your training data, within Amazon S3. The constructor keyword arguments define how SageMaker runs your training script and are discussed, in detail, in a later section.
+
+In the following sections, we'll discuss how to prepare a training script for execution on SageMaker, then how to run that script on SageMaker using an ``MXNet`` Estimator.
+
+Preparing the MXNet training script
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------+
+| WARNING                                                                                                                                                  |
++==========================================================================================================================================================+
+| The structure for training scripts changed with MXNet version 1.3.                                                                                       |
+| Make sure you refer to the correct section of this README when you prepare your script.                                                                  |
+| For information on how to upgrade an old script to the new format, see `"Updating your MXNet training script" <#updating-your-mxnet-training-script>`__. |
++----------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+For versions 1.3 and higher
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Your MXNet training script must be a Python 2.7 or 3.5 compatible source file.
+
+The training script is very similar to a training script you might run outside of SageMaker, but you can access useful properties about the training environment through various environment variables, including the following:
+
+* ``SM_MODEL_DIR``: A string that represents the path where the training job should write the model artifacts to.
+  After training, artifacts in this directory are uploaded to S3 for model hosting.
+* ``SM_NUM_GPUS``: An integer representing the number of GPUs available to the host.
+* ``SM_OUTPUT_DATA_DIR``: A string that represents the path to the directory to write output artifacts to.
```

Comment:
I think this is not true. I just tested this actually. Things stored in '/opt/ml/output' doesn't get uploaded. Only reference I can find about this directory is if you put anything in '/opt/ml/output/failure' will show up in the DescribeTrainingJob call response.

Reply:
but our integ tests checking for a success file in the output dir do pass?

* https://github.com/aws/sagemaker-containers/blob/8b02b03a4edd6bfc838d955993c74a3100b121ab/src/sagemaker_containers/_files.py#L28-L34
* https://github.com/aws/sagemaker-mxnet-container/blob/master/test/integration/__init__.py#L19-L22